modules/nested-accordion/assets/scss/frontend.scss

```
@@ -80,17 +80,16 @@
 
 					span {
 						height: var(--n-accordion-icon-size);
-						width: fit-content;
-						font-size: var(--n-accordion-icon-size);
+						width: auto;
 
 						& > i {
 							color: var(--n-accordion-icon-normal-color);
+							font-size: var(--n-accordion-icon-size);
 						}
 
 						& > svg {
 							fill: var(--n-accordion-icon-normal-color);
-							height: auto;
-							width: inherit;
+							width: var(--n-accordion-icon-size);
```

When you set the address icon to 80px, while the experiment is off, then computed `i` sizes: 
`width: 90px;`
`height: 80px;`

With your styling the SVG icon has the size: 
`width: 80px;`

I am thinking to make sure that the styling doesn't change, when changing the experiment, it might be safer to set the SVG `height` instead of the `width`.

It could be that the PW tests will automatically pass after this change.

```suggestion
							height: var(--n-accordion-icon-size);
```

---

@hein-obox Thanks good suggestion